libraries/plugins/witness/witness.cpp

```diff
@@ -138,9 +138,9 @@ void witness_plugin::plugin_initialize(const boost::program_options::variables_m
       {
          if (fc::exists(key_id_to_wif_pair_file))
          {
-            std::string file_content;
-            fc::read_file_contents(key_id_to_wif_pair_file, file_content);
-            std::istringstream file_content_as_stream(file_content);
+             std::ifstream file_content_as_stream( key_id_to_wif_pair_file.string() );
+             FC_ASSERT( !file_content_as_stream.fail() && !file_content_as_stream.bad(),
+                        "Failed to open file '${f}'", ("f",key_id_to_wif_pair_file.string()) );
```

Comment:
The indentation is not aligned with the code below :P

Reply:
Fixed